e3sm_diags/parser/meridional_mean_2d_parser.py

```diff
@@ -1,10 +1,11 @@
+from e3sm_diags.parameter.meridional_mean_2d_parameter import MeridionalMean2dParameter
 from e3sm_diags.parser.core_parser import CoreParser
 
 
 class MeridionalMean2dParser(CoreParser):
     def __init__(self, *args, **kwargs):
         # FIXME: B026 Star-arg unpacking after a keyword argument is strongly discouraged
-        super().__init__(parameter_cls=MeridionalMean2dParser, *args, **kwargs)  # type: ignore  # noqa: B026
+        super().__init__(parameter_cls=MeridionalMean2dParameter, *args, **kwargs)  # type: ignore  # noqa: B026
```

Comment:
When testing this PR running e3sm_diags on commandline, I found the issue: AttributeError: 'MeridionalMean2dParser' object has no attribute 'check_values'. The MeridionalMean2dParser was incorrectly passing itself as the parameter_cls to CoreParser.__init__() instead of using MeridionalMean2dParameter. This caused the 'check_values' method to be missing, resulting in an AttributeError when running meridional_mean_2d diagnostics. Change here fixed this issue.